src/presence_analyzer/tests.py

```diff
@@ -86,6 +149,78 @@ def test_get_data(self):
             datetime.time(9, 39, 5)
         )
 
+    @patch('presence_analyzer.utils.log')
+    def test_malformed_get_data(self, mock_log):
+        """
+        Test get_data() with malformed data
+        """
+        with patch.dict(main.app.config, {'DATA_CSV': MALF_DATA_CSV}):
+            utils.get_data()
+            self.assertTrue(mock_log.debug.called)
```

Comment:
assert itself can be moved out of with scope,
also, if we're sticking to decorators, we can move it like so:
![image](https://user-images.githubusercontent.com/12252472/54109797-f513e300-43df-11e9-87f8-67e0f4802598.png)